### PR TITLE
Clarify purpose and practice of ensuring interoperability. #120

### PR DESCRIPTION
**Pull request by fantasai**
Addresses https://github.com/w3c/AB-public/issues/120

***
<a href="https://pr-preview.s3.amazonaws.com/fantasai/AB-public/pull/136.html" title="Last updated on Oct 26, 2023, 6:03 AM UTC (dc9455b)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/AB-public/136/61d12dc...fantasai:dc9455b.html" title="Last updated on Oct 26, 2023, 6:03 AM UTC (dc9455b)">Diff</a>

### CODE REVIEW

File: Vision/Vision.bs [hunk 1]
```diff
@@ -159,11 +159,10 @@ Markup Shorthands: markdown yes
...
 		and take feedback from the general public.
-	* **Interoperability**: We believe proven interoperable implementation is a 
-		requirement for broad adoption of standards. 
-		To ensure reliable interoperability, 
-		we require multiple implementations 
-		and open test suites for our standards.
+	* **Interoperability**: We strive to prove the fitness of our specifications
+		through open test suites and actual implementation experience,
+		because we believe the purpose of standards is
+		to enable multiple independent interoperable implementations.
```
Comment (chaals) [inline Vision/Vision.bs:165]: I don't think this clause is necessary - this list is things we do, having explained what we believe elsewhere in the document.

(We could cross-link all these to the rationales, but that makes for a heavier document, which I think is the wrong direction).

Comment (chaals) [inline Vision/Vision.bs:165]:
```suggestion
		through open test suites and actual implementation experience.
```

**Review by chaals — COMMENTED** [review]
Suggested a trim, but 🤷

File: Vision/Vision.bs [hunk 2]
```diff
@@ -159,11 +159,10 @@ Markup Shorthands: markdown yes
 	* **Open Participation**: We welcome individuals and organizations of all sizes
 		(from single-person companies to multi-nationals),
 		and take feedback from the general public.
-	* **Interoperability**: We believe proven interoperable implementation is a 
-		requirement for broad adoption of standards. 
-		To ensure reliable interoperability, 
-		we require multiple implementations 
-		and open test suites for our standards.
+	* **Interoperability**: We strive to prove the fitness of our specifications
```
Comment (cwilso) [inline Vision/Vision.bs:162]: "We strive to prove the fitness of our specifications" is a convoluted phrase, and I particularly wonder whether that makes it harder for an ESL reader to understand.

Comment (fantasai) [reply Vision/Vision.bs:162]: verify sgtm

**Comment by cwilso** [comment]
I tried to add a change, but I think I can't edit your file. I think this bullet should read:

	* **Interoperability**: We verify the fitness of our specifications
		through open test suites and actual implementation experience,
		because we believe the purpose of standards is
		to enable independent interoperable implementations.

(I don't think the word "multiple" is necessary, as it's implied by independent interoperable implementations.)

**Comment by cwilso** [comment]
I pushed the above to #152, PTAL.

**Comment by cwilso** [comment]
replaced by #152

**Pull request closed without merging**